app/controllers/spree/admin/plans_controller.rb

```diff
@@ -30,9 +30,14 @@ def update
           render :edit
         end
       end
-      
+
       def destroy
-        @plan.restrictive_destroy_with_api
+        if @plan.restrictive_destroy_with_api
+         flash[:success] = "Plan has been deleted."
```

Comment:
@praveendhawan I think these are js requests hence we should use `flash.now` instead.